### ISSUE TITLE
Improvements mentioned in sourcecode

### ISSUE THREAD

**Issue by leifdenby** [EUREC4A-UK/lagtraj#95]
Creating trajectories:

- Add metadata to NetCDF output
- Improve linear trajectory to work with haversine functions and actual velocities
- Relax assumption of hourly data?

**Comment by sjboeing** [comment]
I think relaxing the assumption of hourly data may be the trickiest of these to test well. The rest could certainly be part of a first tagged version.

**Comment by sjboeing** [comment]
It looks like we are already using the haversine function in:
extrapolate_posn_with_fixed_velocity

**Comment by leifdenby** [comment]
ok, so I'll make an issue to track the idea of making the improvement to "relax assumption of hourly data"? I don't quite understand what that actually means. We compute the time increments explicitly, so we should be able to support input with different time resolution already, no? Or is this referring to be able to interpolate in time?

**Comment by sjboeing** [comment]
I agree that as long as time increments are explicit everywhere, there should not be a problem in changing the input time resolution. We would need to check if there aren't any explicit references to the hourly character of the data Maybe we can test this by using 3-hourly input data? I would leave any time interpolation to beyond v0.1.0.

**Comment by sjboeing** [comment]
@leifdenby: changing the title as I think the other points (including NetCDF metadata) have now been addressed.
